docs/reference-architectures/virtual-machines-windows/multi-region-application.md

```diff
@@ -104,7 +104,7 @@ Depending on the cause of a failover, you might need to redeploy the resources w
 * Application subsystems are healthy. 
 * Functional testing. (For example, the database tier is reachable from the web tier.)
 
-### SQL Server Always On configuration
+### Configuring the SQL Server Always On Availability Group
```

Comment:
lose the article (the), use the imperative (we are removing -ing from all our headings, according to Christopher)